hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoUpdateHandler.java

```diff
@@ -390,6 +390,8 @@ private Fees cryptoUpdateFees(
         final var keySize = op.hasKey() ? getAccountKeyStorageSize(fromPbj(op.key())) : 0L;
         final var baseSize = baseSizeOf(op, keySize);
         final var newMemoSize = op.memoOrElse("").getBytes(StandardCharsets.UTF_8).length;
+
+        @SuppressWarnings("java:S2259") // account cannot be null after the condition
```

Comment:
Can `account.memo()` be `null`?

Reply:
as I see it cannot
public String memoOrElse(@NonNull final String defaultValue) {
        return hasMemo() ? memo : defaultValue;
    }

Reply:
fixed it thank you